### ISSUE TITLE
correct repo linked

### ISSUE THREAD

**Issue by MaijaLehto** [MaijaLehto/product-page#1]
Hi @wendywarren

Sorry! I didn't even realize I had sent that issue on the profile page instead of product page! too many p's in similar names haha. Is this all you meant by setting up the page? I think everything should be pushed and updated here?

**Comment by wendywarren** [comment]
Hi @MaijaLehto,

You can set up gh pages by clicking on the settings tab and scrolling down to pages. Set your branch to main and then click save.

After you refresh that page and then go back to the code tab in your repo a section for environments should show up on the right of the page.

I cannot shake this fever and cold today so class will be canceled tomorrow. I'm about to send out an email through Brightspace.

If you wouldn't mind describing exactly what you'd like me to look into within this issue thread, I can look at it tomorrow. You won't be penalized if you end up handing it in a bit later.

Sorry. What a crappy way to end the semester with you all.

**Comment by MaijaLehto** [comment]
Alright I think that should be working now? Thank you, I wasn't quite sure what you meant. I'm sorry you're not feeling well either, it was wonderful having you as a professor!

My issue is that I have a div called .top that I'm using to rearrange my images and a chunk of information that goes beside it when i size between tablet and mobile media queries. The information chunk has a large bit of white space beneath it, I'm assuming because it's trying to match the size of the photos, and it's leaving a gap in my website in the mobile format. 
Bryar figured out how to make it smaller so if it can't be figured out by the due time I'm sure it will be fine, but ideally I wanted the photos and chunk of information to be the same length visually in tablet size, and for there to be no space between the end of my .topinfo section and the beginning of the description paragraph below it in mobile.

Thank you, and I hope you feel better soon!

**Comment by wendywarren** [comment]
Thanks @MaijaLehto,

I'm still pretty sick so I'm glad I made that call to cancel class. I did just take a look and noticed that you have two rows set which is probably causing your problem. See screenshot below. 

![Screen Shot 2022-10-19 at 1 41 24 PM](https://user-images.githubusercontent.com/12865609/196765811-8122c17f-e6a4-429b-abb9-a1ef5180421b.png)

Let me know if that's what you're referring to.

Thanks,
Wendy

**Comment by MaijaLehto** [comment]
Yes that did it! What an easy fix... Thank you though, my site is all finished after that! should I make another issue named complete or submission or something?
